test/integration/sanitation-test.coffee

```diff
@@ -48,29 +48,30 @@ describe('Sanitation of Reported Data', ->
       }
     })
 
-  # helper for preparing server
-  createServer = (response) ->
-    app = express()
+  # helper for preparing the server under test
+  createServerFromResponse = (response) ->
+    app = createServer()
     app.put('/resource', (req, res) -> res.json(response))
     return app
 
 
   describe('Sanitation of the Entire Request Body', ->
-    results = undefined
+    dreddRuntimeInfo = undefined
 
     beforeEach((done) ->
-      dredd = createDredd('entire-request-body')
-      app = createServer({name: 123}) # 'name' should be string → failing test
+      dredd = createDreddFromFixture('entire-request-body')
+      app = createServerFromResponse({name: 123}) # 'name' should be string → failing test
 
-      runDreddWithServer(dredd, app, (args...) ->
-        [err, results] = args
-        done(err)
+      runDreddWithServer(dredd, app, (err, runtimeInfo) ->
+        return done(err) if err
```

Comment:
You are usually skipping these returns and sticking to `done(err)`; I'd keep it for the sake of the consistency.